[REF-5952] build: adding a small script to create the component direc…

lior-amar-pm opened the pull request:
…tory

lior-amar committed:
[REF-5952] build: adding a small script to create the component directory

PMjoydas reviewed (commented):
The JIRA # on the commit seems to be incorrect

The new h2o changes for DAI/H2O-3 support will make these changes obsolete

lior-amar-pm commented:
Yes, but when the new stuff is pushed we will need to let the developer know what to fix.

lior-amar committed:
[REF-5952] model-connector: removing model connectors since these are not standard components

o Model connectors moved to internal repo since these are not general components
